platform-tooling-support-tests/src/test/java/platform/tooling/support/tests/ToolProviderTests.java

```diff
@@ -116,7 +116,7 @@ void findAndRunJUnitOnTheModulePath() {
 		var layer = bootLayer.defineModulesWithOneLoader(configuration, ClassLoader.getPlatformClassLoader());
 
 		var sl = ServiceLoader.load(layer, ToolProvider.class);
-		var junit = StreamSupport.stream(sl.spliterator(), false).filter(p -> p.name().equals("junit")).findFirst();
+		var junit = StreamSupport.stream(sl.spliterator(), false).filter(p -> "junit".equals(p.name())).findFirst();
```

Comment:
[tmc] this seems to be leftover/ongoing from:

- https://github.com/junit-team/junit-framework/pull/4748